apps/prairielearn/src/ee/lib/ai-grading/ai-grading-util.ts

```diff
@@ -346,9 +346,7 @@ export async function selectInstanceQuestionsForAssessmentQuestion(
 ): Promise<InstanceQuestion[]> {
   return await queryRows(
     sql.select_instance_questions_for_assessment_question,
-    {
-      assessment_question_id,
-    },
+    { assessment_question_id },
```

Comment:
This was easier to fix in bulk, so I changed it everywhere instead of just the places impacted by the other PR.